tests/test_overlap_module.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+
+from svdb.overlap_module import isSameVariation, precise_overlap
+#from svdb.overlap_module import precise_overlap
```

Comment:
```suggestion
```
Remove unused code

Reply:
Hello!
And thanks for your feedback!
I have removed the unused code and empty lines!

I will continue cleaning up the SVDB merge module

//Jesper

Reply:
Sounds great. Let me know if you need help with something.
Have a nice weekend in a couple of hours :)

/Daniel

Reply:
Thanks! I will make a pull request once I'm done!
It would be great if you could have a look at it as well!

have a nice weekend!